data.json

```
@@ -314,6 +314,15 @@
     "dateAdded": "2018-10-20T12:48:46.000Z",
     "tags": ["hacktoberfest", "expired"]
   },
+  {
+    "name": "Uno",
+    "difficulty": "easy",
+    "description": "Register for the Hacktoberfest® event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
```

```suggestion
    "description": "Register for the Hacktoberfest event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
```
Not sure about the registered trademark symbol.